TestProject/Controllers/PostController.cs

```diff
@@ -69,11 +73,11 @@ public async Task<IActionResult> Edit([FromBody] EditPostModel post)
                 }
                 return Ok();
             }
-            catch
```

Comment:
You should always catch some specific exception. Catching `Exception` is only for edge cases.